karma.conf.js

```diff
@@ -2,203 +2,193 @@ let argv = require('yargs').argv;
-
-            { pattern: 'src/load.js', included: true, served: true },
-
-            { pattern: 'test/test.js', included: true, served: true },
-            { pattern: 'test/**/*', included: false, served: true }
-        ],
-
-        plugins: [
-            require('karma-webpack'),
-            require('karma-mocha'),
-            require('karma-phantomjs-launcher'),
-            require('karma-chrome-launcher'),
-            require('karma-safari-launcher'),
-            require('karma-firefox-launcher'),
-            require('karma-ie-launcher'),
-            require('karma-slimerjs-launcher'),
-            require('karma-sinon-chai'),
-            require('karma-coverage'),
-            require('karma-spec-reporter'),
-            require('karma-sourcemap-loader')
-        ],
-
-        webpackMiddleware: {
-            noInfo: true,
-            stats: false
-        },
-
-        webpack: {
-            devtool: 'inline-source-map',
-
-            resolve: {
-                modules: [
-                    __dirname,
-                    'node_modules'
-                ]
-            },
+    if (karmaConfig.browsers.indexOf('PhantomJS_custom') !== -1) {
+        addPhantom();
+    }
 
-            module: {
-                rules: [
-                    {
-                        test: /\.js$/,
-                        exclude: /(dist|chai)/,
-                        loader: 'babel-loader',
-                        query: {
-                            cacheDirectory: true,
-                            presets: [ [ 'es2015', { 'modules': false } ] ],
-                            plugins: [
-                                'transform-flow-strip-types',
-                                'transform-object-rest-spread',
-                                'syntax-object-rest-spread',
-                                'transform-es3-property-literals',
-                                'transform-es3-member-expression-literals',
-                                'transform-decorators-legacy',
-                                ['transform-es2015-for-of', {loose: true}],
-                                ['flow-runtime', {
-                                    'assert': true,
-                                    'annotate': true
-                                }],
-                                [ 'istanbul', { only: `${__dirname}/src` } ]
-                            ]
-                        }
-                    },
-                    {
-                        test: /\.(html?|css|json)$/,
-                        loader: 'raw-loader'
-                    }
-                ]
+}
+
+function addPhantom() {
+    karmaConfig.phantomjsLauncher = {
+        exitOnResourceError: true
+    };
+    karmaConfig.customLaunchers.PhantomJS_custom = {
+        base: 'PhantomJS',
+        options: {
+            windowName: 'my-window',
+            settings: {
+                webSecurityEnabled: false
```

Comment:
Can we leave web security enabled? Or do you see problems with it. Tests pass for me with this on, and I'd prefer to be closer to production settings.